src/main/java/com/hms_networks/sc/cumulocity/api/CConnectorMqttMgr.java

```diff
@@ -368,10 +372,22 @@ public void sendConfigurationFileToCumulocity() {
    */
   public void sendMessageWithChildDeviceRouting(String messagePayload, String childDevice)
```

Comment:
add comment to javadoc -- also adds chidDevice to list of registeredChildDevices

Reply:
Added